mimic/model/identity.py

```diff
@@ -20,3 +26,205 @@ def from_store(cls, session_store, tenant_id):
 
     identity_admin = attr(default=False,
                           validator=validators.instance_of(bool))
+
+
+class ICredentials(Interface):
+    """
+    An :obj:`ICredentials` provides identity authentication credentials.
+    """
+    def get_session(session_store):  # pragma:nocover
+        """
+        Get a session corresponding to the user and tenant from the given
+        session store.
+
+        :param session_store: a :class:`mimic.session.SessionStore`
```

Comment:
Given the perpetual confusion in the area of "what even is a Session", a `:return:` or `:rtype:` would definitely not be misplaced.